src/Commands/FindMissingTranslations.php

```diff
@@ -11,7 +11,7 @@
 /**
  * Inspired by https://github.com/VetonMuhaxhiri/Laravel-find-missing-translations
  */
-class FindMissingTranslations extends Command
+final class FindMissingTranslations extends Command
```

Comment:
it's considered as a BC break (some users may extend this console command). Can you please make it non-final again? Thank you! If it's Psalm that asks to finalize classes, you can add `@api` PHPDoc to the class.

Reply:
I set it to final to fix psalm error, anyway I reverted the change and added `@api`